consensus/common/src/main/java/org/hyperledger/besu/consensus/common/bft/headervalidationrules/BftValidatorsValidationRule.java

```diff
@@ -37,6 +37,15 @@
 public class BftValidatorsValidationRule implements AttachedBlockHeaderValidationRule {
 
   private static final Logger LOGGER = LogManager.getLogger();
+  private final boolean extraDataValidatorsAndVoteMustBeEmpty;
+
+  public BftValidatorsValidationRule() {
+    this(false);
+  }
+
+  public BftValidatorsValidationRule(final boolean extraDataValidatorsAndVoteMustBeEmpty) {
```

Comment:
This rule is quite specific to qbft and so I don't think it should be part of the common rules. Would rather the common bft code didn't know anything about contract based voting/validator governance.

Reply:
refactored.